internal/sgbouncewizard/sgbouncewizard.go

```diff
@@ -264,6 +267,14 @@ func (srv *Server) DeleteRuleRoute(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	log.WithFields(log.Fields{
```

Comment:
I'd also add the deleted ID to the log fields and just make the message constant. That way you can see the entire lifecycle of, for example, rule 312 by searching through your log aggregator for `RuleID=312`. You also then don't have to do string comparison across the entire `message` portion of the log. (These are probably overkill for this system, but for larger systems it's good to think about how logs can be indexed).